en-us/Content/index.htm

```diff
@@ -288,7 +288,7 @@
                                                     <h1 style="margin-top:35px;">Okta Documentation</h1>
                                                     <div class="tiles">
                                                         <div>
-                                                            <h3 class="tile-title"><a href="https://help.okta.com/oie/en-us/Content/Topics/identity-engine/oie-index.htm">Okta Identity Engine</a>
+                                                            <h3 class="tile-title"><a href="../../oie/en-us/Content/Topics/identity-engine/oie-index.htm">Okta Identity Engine</a>
```

Comment:
Can we use absolute path? Such as `/oie/en-us/...` instead of `../../oie/en-us/...`

Reply:
we're going to revisit this later, looking into testing implications of relpath vs. abspath